content/applications/hr/lunch.rst

```diff
@@ -0,0 +1,313 @@
+     available to select from the list.
+
+- :guilabel:`Send Order By`: click the radio button to select how orders are sent to the vendor. The
+  available options are :guilabel:`Phone` or :guilabel:`Email`.
+- :guilabel:`Order Time`: this field appears if :guilabel:`Email` is selected for :guilabel:`Send
+  Order By`. Enter the time that an order must be emailed for the order to be accepted. Enter the
+  time in a `##:##` format, then select either :guilabel:`AM` or :guilabel:`PM` from the drop-down
+  next to the time field.
+
+.. image:: lunch/orders.png
+   :align: center
+   :alt: The orders section of a vendor form, with all fields filled out.
+
+Extras
+------
+
+When ordering an item in the *Lunch* app, it is possible to show extra items that can be added to an
+order, sometimes referred to as *add-ons*. These can be configured in any manner that suits the
+products being offered.
+
+By default, Odoo allows for three types of extra items, which can be thought of as *categories*. By
+default, the first type or *category* of add-ons is labeled `Extras`, the second is labeled
+`Beverages`, and the third is labeled `Extra Label 3`.
+
+.. important::
+   When configuring the extras, it is important to keep in mind that all the extras configured will
+   appear for **every item** offered by the vendor. That means that only items which apply to
+   **all** products from the vendor should be added.
+
+Configure Extras
+~~~~~~~~~~~~~~~~
+
+Enter the following information for each of the three available extra sections:
+
+- :guilabel:`Extra (#) Label`: enter a name for the type of extra, such as `Toppings` or
+  `Beverages`. This can be thought of as a *category*.
+- :guilabel:`Extra (#) Quantity`: select how the extras are allowed to be selected. The options are:
+
+  - :guilabel:`None or More`: select this if the user is not required to select anything.
+  - :guilabel:`One or More`: select this to require the user to make at least one selection.
+  - :guilabel:`Only One`: select this if the user can only make one selection.
+
+Add Extras
+~~~~~~~~~~
+
+After the labels and quantities have been configured for an extra category, the individual extra
+items must be added for each category.
+
+Click :guilabel:`Add a line` at the bottom of the list that appears on the right-hand side of the
+extra category. Enter the :guilabel:`Name` and :guilabel:`Price` for each item being added. The
+price can remain at `$0.00` if there is no cost. This is common for items like disposable silverware
+or condiments.
+
+.. example::
+   For a pizzeria that only offers personal pies has their extras configured as follows:
+
+   The first extra is configured for the various toppings they offer. The :guilabel:`Extra 1 Label`
+   is set to `Toppings`, and the :guilabel:`Extra 1 Quantity` is set to :guilabel:`None or More`.
+   The various toppings are then added, with their corresponding costs:
```

Comment:
Use a period here since images shouldnt be used as part of an instruction in place of text
```suggestion
   The various toppings are then added, with their corresponding costs.
```